CHANGELOG.md

```diff
@@ -64,18 +64,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * AVIF files with an exif_tiff_header_offset pointing at another byte than the
   first II or MM tag in the Exif metadata payload will now fail to be decoded.
   Set decoder->ignoreExif to true to skip the issue and decode the image.
-* Fix memory errors reported in crbug.com/1501766 and crbug.com/1501770.
+* Fix memory errors reported in crbug.com/1501766, crbug.com/1501770, and
+  crbug.com/1504792 by [Fudan University](https://secsys.fudan.edu.cn/).
 * For codecs, AVIF_CODEC_* and AVIF_LOCAL_* are now merged into AVIF_CODEC_*
   that can only take the values: OFF, LOCAL or SYSTEM.
 * For the libyuv, libsharpyuv, zlibpng and jpeg dependencies, AVIF_LOCAL_* is
   now replaced by flags AVIF_* that can take the values: OFF, LOCAL or SYSTEM.
+* src/reformat.c: Allocate the threadData array directly.
```

Comment:
Note that I repeated the v1.0.3 bug fixes in the Changed section of the next release from the main branch, because that release will diff against the v1.0.0 release.

Reply:
Should the `### Changed` titles be more specific then? Such as

```md
### Changed since v1.0.0
```

Reply:
Yes. For some people, this is expected (that the v1.1.0 changelog is with respect to v1.0.0), but it is good to be explicit.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2017.